source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRTextView.mm

```diff
@@ -142,15 +143,25 @@ - (void)textViewDidBeginEditing:(UITextView *)textView
 {
     if (_placeholderText && [textView.text isEqualToString:_placeholderText]) {
         textView.text = @"";
-        textView.textColor = [UIColor blackColor];
+        if (@available(iOS 13.0, *)) {
```

Comment:
We should test both on iOS13 as well as previous versions - both to ensure no regressions.

Reply:
Additionally once this is in, do remember to cherry pick into release/1.2 branch.. thanks!